x-pack/plugins/apm/server/lib/helpers/setup_request.test.ts

```diff
@@ -19,28 +19,36 @@ describe('setupRequest', () => {
         config: () => 'myConfig',
         plugins: {
           elasticsearch: {
-            getCluster: () => ({
-              callWithRequest: callWithRequestSpy
-            })
+            getCluster: () => ({ callWithRequest: callWithRequestSpy })
           }
         }
-      }
+      },
+      getUiSettingsService: jest.fn(() => ({
+        get: jest.fn(() => Promise.resolve(false))
+      }))
     };
-
-    const setup = setupRequest(mockReq);
-    setup.client('myType', { body: 'foo' });
   });
 
-  it('should call callWithRequest with correct args', () => {
+  it('should call callWithRequest with correct args', async () => {
+    const setup = setupRequest(mockReq);
+    await setup.client('myType', { body: 'foo' });
     expect(callWithRequestSpy).toHaveBeenCalledWith(mockReq, 'myType', {
       body: 'foo',
+      ignore_throttled: true,
       rest_total_hits_as_int: true
     });
   });
 
-  it('should update params with rest_total_hits_as_int', () => {
```

Comment:
This test didn't appear to be testing anything different than the "with correct args" test above.